Source/Core/Common/FileUtil.cpp

```diff
@@ -226,6 +229,27 @@ bool CreateFullPath(const std::string& fullPath)
   }
 }
 
+// Returns a human-readable string representing a filesize
+std::string PrettyPrintFileSize(u64 size)
```

Comment:
Fair enough, do you have a recommendation for where it should go? I'm thinking something like DiscIO/Filesystem.cpp